lib/udap_security_test_kit.rb

```diff
@@ -57,6 +57,21 @@ class Suite < Inferno::TestSuite
       request.query_parameters['state']
     end
 
+    links [
+      {
+        label: 'Report Issue',
+        url: 'https://github.com/inferno-framework/udap-security-test-kit/issues'
+      },
+      {
+        label: 'Open Source',
+        url: 'https://github.com/inferno-framework/udap-security-test-kit'
+      },
+      {
+        label: 'Download',
+        url: 'https://github.com/inferno-framework/udap-security-test-kit/releases'
+      }
```

Comment:
I think we want to do 'Implementation Guide'?  Pretty sure that's the specific link that was being looked for in this case, though I could be misremembering